.gitignore

```diff
@@ -1 +1,3 @@
 /bin/
+upi/metal/matchbox/
+upi/metal/terraform.tfvars
```

Comment:
currently, we only keep the artifacts created by our build scripts in the .gitignore and therefore the upi/** changes should not be added to this file. see https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally for adding these to your local repo

Reply:
Ok I'll remove that change

Reply:
Removed 8d3bd1702cf751fadf5018408fa9761a005921b3 with that change